aws/resource_aws_cognito_user.go

```diff
@@ -0,0 +1,284 @@
+package aws
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAwsCognitoUser() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsCognitoUserCreate,
+		Read:   resourceAwsCognitoUserRead,
+		Update: resourceAwsCognitoUserUpdate,
+		Delete: resourceAwsCognitoUserDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAwsCognitoUserImport,
+		},
+
+		// https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminCreateUser.html
+		Schema: map[string]*schema.Schema{
+			"user_attribute": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"value": {
+							Type:     schema.TypeString,
+							Optional: true,
+							// Sensitive: true,
```

Comment:
https://github.com/aws/aws-sdk-go/blob/v1.38.64/service/cognitoidentityprovider/api.go#L15444-L15454

I assume the field should be marked sensitive here as well then